content/java/concepts/hashset/hashset.md

```diff
@@ -0,0 +1,193 @@
+---
+Title: 'HashSet'
+Description: 'A HashSet stores a set of items that are unique, unordered, and of a given datatype.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Data Structures'
+  - 'Data Types'
+  - 'Collections'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+A `HashSet` is used to store an unordered set of unique items of a given [data type](https://www.codecademy.com/resources/docs/java/data-types).
+
+## Syntax
+
+```java
+import java.util.HashSet;
+
+HashSet<DataType> h = new HashSet<DataType>();
+```
+
+Where `h` is a `HashSet` created with the `new` keyword and `DataType` is the data type that will be stored in it.
+
+**Note:** `HashSet` is a member of the `java.util` package which must be imported first.
```

Comment:
Using the Java [Hashmap](https://www.codecademy.com/resources/docs/java/hashmap) entry as an example, I'm thinking that instead of this note, maybe we can incorporate this into the explanation above